metagraph/src/annotation/row_diff_builder.cpp

```diff
@@ -402,6 +405,9 @@ void convert_batch_to_row_diff(const std::string &pred_succ_fprefix,
                 }
             },
             [&](uint64_t block_begin) {
+                if (!compute_row_reduction)
```

Comment:
Isn't it more efficient to just pass an empty lambda if `compute_row_reduction` is false?